projects/extension/public/manifest.json

```diff
@@ -31,6 +32,8 @@
     "48": "./icons/icon-48.png",
     "128": "./icons/icon-128.png"
   },
-  "content_security_policy": "script-src 'self' 'wasm-unsafe-eval'; object-src 'self'",
-  "web_accessible_resources": ["page.js"]
+  "content_security_policy": {
+    "script-src": "self",
```

Comment:
we need the `wasm-unsafe-eval` in the content_security_policy. Stores will not accept the extension without that (happened before)